src/domain/gpqueue.ts

```diff
@@ -113,7 +115,16 @@ export function create(dao: Address,
                     genericSchemeMultiCall.voteParams(),
                     genericSchemeMultiCall.schemeConstraints());
     isGPQue = true;
-  }
+  } else if (equalStrings(contractInfo.name, 'ContinuousLocking4Reputation')) {
+    let continuousLocking4Reputation = ContinuousLocking4Reputation.bind(scheme);
```

Comment:
it is a bit weird that this is here..because it has no genesis protocol ..
is that can be moved to other place ?

Reply:
It's problematic... we could move it to the index code but that's uglier, I think here is best since all schemes are initialized there